packages/components/src/components/alert/alert.custom.styles.ts

```diff
@@ -43,10 +48,11 @@ export default css`
   .alert__icon,
   .alert__close-button {
     align-items: flex-start;
-    font-size: var(--syn-font-size-x-large);
+    font-size: var(--size-icon-size);
   }
 
   .alert__message {
+    justify-content: center;
```

Comment:
This needs to be `align-self: center;`
Justify-content is not event set into account

<img width="419" height="149" alt="Image" src="https://github.com/user-attachments/assets/44af03be-5b17-49ec-bbff-5e7c82c1a1ea" />


```suggestion
    align-self: center;
```